make/test.mk

```diff
@@ -6,7 +6,7 @@
 
 .PHONY: test
 ## runs the tests without coverage and excluding E2E tests
-test:
+test: generate
```

Comment:
generate is maybe a bit confusing? i wouldn't guess by the name that it is running tests

Reply:
actually, it's a dependency goal to execute before the running the tests. And `generate` is an "umbrella" goal for multiple smaller tasks